subsys/bluetooth/controller/ll_sw/ll_filter.c

```diff
@@ -421,7 +421,7 @@ bool ctrl_rl_allowed(u8_t id_addr_type, u8_t *id_addr, u8_t *rl_idx)
 		}
 	}
 
-	return false;
+	return true;
```

Comment:
No, rl_idx is passed as a pointer in case it needs to be updated from no match (IRK) to match (ID address), but if there is no match then it can remain as-is.